files/ansible/tasks/install_perf.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Install perf on Debian
+  apt:
+    name: linux-perf
+    state: present
+    update_cache: yes
+  when: ansible_facts.os_family == 'Debian'
+
+- name: Install perf on Ubuntu
+  apt:
+    name: "{{ item }}"
+    state: present
+    update_cache: yes
+  with_items:
+    - linux-tools
+    - linux-tools-generic
+    - linux-tools-common
```

Comment:
Do we really need all of those packages? If we only need linux-tools, then we can also join this with Arch.